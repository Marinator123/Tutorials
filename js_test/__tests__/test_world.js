import React from 'react'
import renderer from 'react-test-renderer';
import World from '../src/world.jsx';


test('Test example', () => {
    const component = renderer.create(
        <World/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});