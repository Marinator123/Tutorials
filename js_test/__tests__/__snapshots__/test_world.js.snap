// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test example 1`] = `
<h1>
  World
</h1>
`;
